### DISCUSSION TITLE
324 - Unsupported raw media category

### DISCUSSION THREAD

**Discussion by metalaureate** [tweepy/tweepy#1839] [Questions] [answered]
I am using the chunked media upload API to upload a largish (40MB) mp4 to attach to a tweet using Tweepy. But Tweepy returns `Unsupported raw media category` (with or without the `media_category` attribute set to dm_video). 

The problem does not occur if I use a smaller file and the API.media_upload with no media category set, but that returns 400 error if I upload my 40MB file. Media Category is mandatory for chunked uploads.

After some investigation, I am wondering if Tweepy has inherited his defect from twit: https://twittercommunity.com/t/tweeting-w-media-fails-with-error-code-324-unsupported-raw-media-category/106868/15

**Comment by Harmon758** [comment, marked as answer]
If you're attaching it to a Tweet, you should pass `tweet_video` for `media_category` instead.
The forum thread you linked also relates to passing `tweet_video` instead of `dm_video`.

**Comment by metalaureate** [comment]
Hey thanks. I was just about to delete my question having realized this, thank you for such a rapid response!
